PeerioTechnologies/peerio-mobile
# fix: disable share from peerio when filestore is empty [branch ch8198]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8198/mobile-disable-the-share-from-peerio-option-if-the-users-has-0-files
#### Testing instructions  
Case 1:
Create a new account (or use an account with an empty file store). Open a DM with someone. Click on the "+" button in chat to pull up the action sheet. "Share from peerio files" should be disabled.

Case 2:
Using the same account, go to files list. Upload a file.  Open a DM with someone. Click on the "+" button in chat to pull up the action sheet. "Share from peerio files" should be enabled.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
